app/src/main/java/org/eyeseetea/malariacare/data/remote/PullDhisSDKDataSource.java

```diff
@@ -9,20 +9,27 @@
 import org.eyeseetea.malariacare.domain.exception.NetworkException;
 import org.hisp.dhis.client.sdk.android.api.D2;
 import org.hisp.dhis.client.sdk.core.common.controllers.SyncStrategy;
+import org.hisp.dhis.client.sdk.core.event.EventFilters;
 import org.hisp.dhis.client.sdk.models.attribute.Attribute;
+import org.hisp.dhis.client.sdk.models.attribute.AttributeValue;
+import org.hisp.dhis.client.sdk.models.event.Event;
 import org.hisp.dhis.client.sdk.models.organisationunit.OrganisationUnit;
 
+import java.util.ArrayList;
 import java.util.List;
 
 import rx.Observable;
+import rx.Scheduler;
 import rx.android.schedulers.AndroidSchedulers;
 import rx.functions.Action1;
 import rx.functions.Func2;
 import rx.schedulers.Schedulers;
 
 public class PullDhisSDKDataSource {
 
-    public void pullMetadata(final IDataSourceCallback<Void> callback) {
+    public static final String USER_CC_CO_VOL = "USER_CC_CO_VOL";
```

Comment:
@xurxodev 
Is this variable the attribute code of the users?
can you rename it to USER_CATEGORY_COMBINATION_ATTRIBUTE__CODE or some similar more expressive?
thanks!